### DISCUSSION TITLE
Google drive and Dropbox in file manager

### DISCUSSION THREAD

**Discussion by yasnel** [elementary/triage#472] [Ideas]
It would be very useful to have the possibility of connecting to Google Drive or Dropbox from the file manager, it is a function that I have not been able to implement in eOS.

Thanks

**Comment by jeremypw** [comment]
There used to be a separate Dropbox plugin about it was replaced by a core cloudprovider plugin which will create a sidebar item for every client implementing a libcloudprovider DBus interface (see https://world.pages.gitlab.gnome.org/libcloudproviders/).  Unfortunately only NextCloud has a client implementing that interface at the moment afaik. See the Files Wiki for how to install NextCloud.

It would certainly be welcome if a suitable clients for other commercial services became available but I do not think it is within the scope of this project to provide them.
